src/lib/hooks/use-best-quote/utils/issuance/index.ts

```diff
@@ -18,11 +21,22 @@ interface RedemptionQuoteRequest {
   slippage: number
 }
 
+const contract = '0x04b59F9F09750C044D7CfbC177561E409085f0f3'
+
 export async function getEnhancedRedemptionQuote(
   request: RedemptionQuoteRequest,
-  publicClient: PublicClient
+  publicClient: PublicClient,
+  // Using ethers signer for now but will refactor gas estimator to use viem soon
+  signer: providers.JsonRpcSigner
```

Comment:
Will be a separate PR at some point - as I currently can't estimate the impact of updating gas estimator now.